src/isaac-data-types.d.ts

```diff
@@ -570,6 +585,9 @@ export interface LocalUserCredential {
 }
 
 export interface RegisteredUser extends AbstractSegueUser {
+  /**
```

Comment:
are we actually marking fields as deprecated in the API?

Reply:
Yep

 @JsonProperty("_id")
  @Deprecated
  public Long getLegacyId() {
    return this.getId();
  }